src/components/Sum.spec.jsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { mount } from '@cypress/react';
-import { Sum } from './Sum';
+import Sum from './Sum';
```

Comment:
named exports are preferred
```suggestion
import { Sum } from './Sum';
```